tests/main/remodel/task.yaml

```diff
@@ -61,35 +72,42 @@ execute: |
         exit
     fi
 
+    #shellcheck source=tests/lib/systems.sh
+    . "$TESTSLIB"/systems.sh
+    SNAP=test-snapd-tools
```

Comment:
You could set SNAP to `test-snapd-tools-coreXX` to make this test "automatically" start doing stuff on core20. Not a blocker.

Reply:
@zyga In fact, it is a really good idea, we could call systems and this give as the sufix depending on the core system. I'll create a following PR with this change which will impact on many tests. 
Thanks for the suggestion.